### ISSUE TITLE
Improve `parseXML` to use `expandRefenrece`

### ISSUE THREAD

**Issue by mtuchi** [OpenFn/adaptors#447]
### Summary
Modify `parseXML` arguments to use `expandReference` 

### Todo's
- [x] Use `expandReference` on `parseXML`  arguments. Done https://github.com/OpenFn/adaptors/pull/448
- [ ] Add example of traversing the DOM

**Comment by aleksa-krolls** [comment]
@mtuchi how do you think this should be prioritized? Is there still adaptor work to be done here before we can work on development for a new Water Aid workflow?

**Comment by mtuchi** [comment]
@aleksa-krolls the important part of this issue is already done in https://github.com/OpenFn/adaptors/pull/448. And we already cut a new release for http adaptor `v5.1.1` which unblock the testing for water-aid. The remaining task `Add example of traversing the DOM` i would say it's low priority. I think it can be part of the new http adaptor once when we cut a release for the new http adaptor.
